Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -110,7 +110,9 @@ open class ChatChannelVC: _ViewController,
     private var hasSeenLastMessage: Bool = false
 
     /// The id of the first unread message
-    private var firstUnreadMessageId: MessageId?
+    private var firstUnreadMessageId: MessageId? {
+        channelController.firstUnreadMessageId
```

Comment:
This can be problematic. We are switching back to not having local state. This interferes with what I am working on. We might need alignment.

Plus getting `channelController .firstUnreadMessageId` is not cheap